package/MDAnalysis/analysis/contacts.py

```diff
@@ -146,14 +144,15 @@
 import errno
 import warnings
 import bz2
-from six.moves import zip
+from itertools import izip
```

Comment:
the six import was correct. It does import the iterator version of zip under python2 as zip. So no more izip. This is part of our effort to support python3 along with python2 in the future.